aea/test_tools/test_cases.py

```diff
@@ -224,12 +252,71 @@ def run_install(self):
         """
         self.run_cli_command("install")
 
+    def generate_private_key(self) -> None:
+        """
+        Generate AEA private key with CLI command.
+        Run from agent's directory.
+
+        :return: None
+        """
+        self.run_cli_command("generate-key", FETCHAI_NAME)
+
+    def add_private_key(self) -> None:
```

Comment:
add-key should have kwargs: identifier: str, key_file_path: str  rather than hard coded

Reply:
Didn't have a real use cases in tests yet. But I can do that. Will add those args with default values.

Reply:
You'll need to think a bit ahead as well. The code in the `aea/` should be reusable. Your code is too specific for no good reason. providing the args from the start means its versatile and you do not have to go back to it once a slightly different test case comes up.